docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst

```diff
@@ -318,23 +318,23 @@ You can also look at the backup files:
 If `ansible-playbook` fails, please follow the debug steps in :ref:`network_debug_troubleshooting`.
 
 
-.. _network-agnostic-examples:
+.. _network-independent-examples:
 
-Example 2: simplifying playbooks with network agnostic modules
-==============================================================
+Example 2: simplifying playbooks with network independent modules
```

Comment:
This doesn't quite have the same meaning in my ear, plus I could see it being hard to translate. I'm not sure what to suggest, though. Maybe "multi-network" or "multi-vendor"?

Reply:
Reading the content, I think platform independent works well. The point is that it is a generic 'cli_command' module that can work with multiple vendor products.

Reply:
+1 and thanks for adding the hyphen, I think that will help with translation